db-tests/src/test/java/gov/nasa/jpl/aerie/database/PlanCollaborationTests.java

```diff
@@ -3542,4 +3542,48 @@ void tagsAreNotShuffledDuringMerge() throws SQLException {
       assertEquals(case8Tags, tagsHelper.getTagsOnActivity(case8Id, planId));
     }
   }
+
+  @Nested
+  class ExternalEventTests {
```

Comment:
Since the behavior of ExternalEvents interacting with plan merge has been removed, these tests should be removed